src/nccl_ofi_system.c

```diff
@@ -27,6 +27,11 @@ const char *nccl_net_ofi_get_product_name(void)
 	static char *product_name = NULL;
 	static pthread_mutex_t product_name_mutex = PTHREAD_MUTEX_INITIALIZER;
 
+	char* forced_pn = getenv("OFI_NCCL_FORCE_PRODUCT_NAME");
+	if (forced_pn != NULL) {
+		return forced_pn;
+	}
+
```

Comment:
Should this not be done from the util?

Reply:
https://github.com/aws/aws-ofi-nccl/blob/eade05a6f11eee6921aa9f4fe7cf4529b4b88d51/contrib/python/ofi_nccl/tuner/cli/main.py#L20 it does